app/stores/BaseStore.ts

```diff
@@ -61,7 +57,7 @@ export default abstract class BaseStore<T extends BaseModel> {
   constructor(rootStore: RootStore, model: Class<T>) {
     this.rootStore = rootStore;
     this.model = model;
-    this.modelName = modelNameFromClassName(model.name);
+    this.modelName = lowerFirst(model.name).replace(/\d$/, "");
```

Comment:
We’re still replacing digits here, is that still needed?

Reply:
It might not be, was reproducible locally